bolt-sidecar/src/state/execution.rs

```diff
@@ -146,10 +145,21 @@ pub struct ExecutionState<C> {
     /// The cached account states. This should never be read directly.
     /// These only contain the canonical account states at the head block,
     /// not the intermediate states.
-    account_states: HashMap<Address, AccountState>,
+    ///
+    /// The value of the map is a tuple containing the account state and a score.
+    /// The score is needed to determine which accounts to evict when the cache is full with a
+    /// custom logic.
+    /// When a commitment request is made from an account its score is bumped of
+    /// [ACCOUNT_STATE_SCORE_BUMP], and when it updated it is decreased by
+    /// [ACCOUNT_STATE_UPDATE_PENALTY].
```

Comment:
These values seem to not be defined

Reply:
Thanks, and sorry again for all recurring stale docs lately